### PR TITLE
[ARITH][BACKPORT-0.6] fix a min/max simplify bug

### PR DESCRIPTION
**Pull request by xqdan**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

### CODE REVIEW

**Comment by xqdan** [comment]
@tqchen @yzhliu please review

**Comment by tqchen** [comment]
cc @yzhliu @yongfeng-nv @Hzfengsy @merrymercy

File: src/arith/rewrite_simplify.cc [hunk 1]
```diff
@@ -1026,7 +1026,9 @@ PrimExpr RewriteSimplifier::Impl::VisitExpr_(const MinNode* op) {
       int64_t c1val = c1.Eval()->value;
       int64_t c2val = c2.Eval()->value;
       if (c2val % c1val == 0) {
-        if (c2val / c1val >= 0) {
+        if (c2val == 0) {
+          return c1val > 0 ? (min(x, 0) * c1val).Eval() : (max(x, 0) * c1val).Eval();
+        } else if (c2val / c1val > 0) {
```
Comment (tqchen) [inline src/arith/rewrite_simplify.cc:1031]: Thanks @xqdan this rule seems also should applies to other cases when c1val is smaller than 0, 

e.g. (  `min(x * (-2),  -4)`), please also fix that case

Comment (tqchen) [reply src/arith/rewrite_simplify.cc:1031]: We should also fix the case for max

Comment (tqchen) [reply src/arith/rewrite_simplify.cc:1031]: We can change original line from `c2val / c1val >= 0` to `c1val > 0`.

Comment (tqchen) [reply src/arith/rewrite_simplify.cc:1031]: Let us also fix the (rare) special case, `(c1val == 0)`

Comment (xqdan) [reply src/arith/rewrite_simplify.cc]: ok, fixed

**Comment by tqchen** [comment]
Thanks @xqdan ! Please send another PR to the v0.6 branch

**Pull request merged**